### PR TITLE
Update Documentation;

### PR DESCRIPTION
**Pull request by David-Henner**
Updated the README --> https://github.com/conichiGMBH/conichi-ios-sdk/tree/CNI-8942

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,14 +73,21 @@ Following the links below you can find the instructions in the __Getting Started
...
+ <string>This app needs your location so you can be recognized in conichi Hotels</string>
+<key>NSLocationUsageDescription</key>
+ <string>This app needs your location so you can be recognized in conichi Hotels</string>
+<key>NSLocationWhenInUseUsageDescription</key>
+ <string>This app needs your location so you can be recognized in conichi Hotels</string>
 ```
 
-Be careful, if you use `NSLocationWhenInUseUsageDescription` instead, The app may not work properly.
+Be careful, if you decide not to use all the keys, The app may not work properly.
+If you wish to know why those keys are necessary, you can check out [this link](https://github.com/conichiGMBH/conichi-ios-sdk/blob/master/Docs/LocationPermission.md)
```
Comment (ShengHuaWu) [inline README.md]: Perhaps, you can take the advantage of relative path for the link url, for example, `Docs/LocationPermission.md`. 😄

File: README.md [hunk 2]
```diff
@@ -88,11 +97,9 @@ Conichi cares about the security and therefore the SDK stores the authorization
 
 ![Alt text](https://monosnap.com/file/Fgjm7ziUSjarcbDIRgJshGvXUiElh3.png)
 
-And you're off! Take a look at the public documentation and start building.
-
 #### Initialization
 
-Example below shows the simplest setup for `conichiSDK CoreKit` that points to our `sandbox` server.
+Example below shows the simplest setup for `conichiSDK CoreKit` that points to our `development` server.
```
Comment (ShengHuaWu) [inline README.md]: If we are going to allow users to access `development` environment, it's better to mention adding `Allow Arbitrary Loads` into `Info.plist` inside `Update info plist`.
What do you think?

Comment (David-Henner) [reply README.md]: Updated in 3884ca1

File: README.md (hunk 1 above)
Comment (David-Henner) [reply README.md]: Updated in 3884ca1

**Pull request merged**